### ISSUE TITLE
ReferenceError: EventState is not defined

### ISSUE THREAD

**Issue by sebastian-ruiz** [Meteor-Community-Packages/ground-db#215]
I get the following errors when I try and use ground-db with `METEOR@2.10.0`:
```
This package is no longer maintained and have been deprecated.
    <anonymous> http://localhost:3000/packages/raix_eventstate.js?hash=d82aacc8d5e865e27681351e6dbd0ac9f570fca3:26

Exception from Tracker recompute function:
ReferenceError: EventState is not defined

GroundCollection@http://localhost:3000/packages/ground_db.js?hash=0bc7480ec375e5365615382c4bcd8aa22c89751a:143:5
module/</HomePage/addresses<@http://localhost:3000/app/app.js?hash=e5e9170b74dedb22ae68f62fe0453b0e9d2082ae:408:27
module/useTrackerNoDeps/</<@http://localhost:3000/packages/react-meteor-data.js?hash=f9edbe3db490cb06d2d717c3da48420338178c6b:364:28
module/_compute/<@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:310:40
module/Tracker.withComputation@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:585:12
_compute@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:309:15
_recompute@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:324:16
module/Tracker._runFlush@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:481:14
onGlobalMessage@http://localhost:3000/packages/meteor.js?hash=3fe70c93a2fcd225dc5cc6572b8c3e39756a507b:552:23
```

On the branch `remove-eventstate` https://github.com/Meteor-Community-Packages/ground-db/commit/e51ae2f8a7afbeb0553494ab9327bd3e9a840828 I get:

```
Exception from Tracker recompute function:
TypeError: missing argument 0 when calling function Array.prototype.find

module/</HomePage/addresses<@http://localhost:3000/app/app.js?hash=e5e9170b74dedb22ae68f62fe0453b0e9d2082ae:409:49
module/useTrackerNoDeps/</<@http://localhost:3000/packages/react-meteor-data.js?hash=f9edbe3db490cb06d2d717c3da48420338178c6b:364:28
module/_compute/<@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:310:40
module/Tracker.withComputation@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:585:12
_compute@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:309:15
_recompute@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:324:16
module/Tracker._runFlush@http://localhost:3000/packages/tracker.js?hash=1416fd47879241b1300a4665b10cd76adc1484ae:481:14
onGlobalMessage@http://localhost:3000/packages/meteor.js?hash=3fe70c93a2fcd225dc5cc6572b8c3e39756a507b:552:23
EventListener.handleEvent*usePostMessage@http://localhost:3000/packages/meteor.js?hash=3fe70c93a2fcd225dc5cc6572b8c3e39756a507b:561:12
@http://localhost:3000/packages/meteor.js?hash=3fe70c93a2fcd225dc5cc6572b8c3e39756a507b:590:3
@http://localhost:3000/packages/meteor.js?hash=3fe70c93a2fcd225dc5cc6572b8c3e39756a507b:595:4
@http://localhost:3000/packages/meteor.js?hash=3fe70c93a2fcd225dc5cc6572b8c3e39756a507b:1417:3
```

**Comment by raix** [comment]
I've moved over the EventState package https://github.com/Meteor-Community-Packages/Meteor-EventState

**Comment by macrozone** [comment]
hi @raix and @sebastian-ruiz did you find a solution to this?

**Comment by StorytellerCZ** [comment]
This will be fixed in next patch release where this dependency is removed.

**Comment by mattsinnock** [comment]
Hi folks, is there a workaround for this? I'm attempting to upgrade my app which uses ground:db to Meteor 2.16 and I'm hitting this same error.

**Comment by StorytellerCZ** [comment]
Released 2.1.0, this should now be fixed.
